.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: cpp
+
+os: linux
+dist: trusty
+sudo: required
+
+before_install:
+  - deps/jerry/tools/apt-get-install-deps.sh
+
+install:
+
+script: "tools/build.py $OPTS"
+
+env:
+  - OPTS="--buildtype=debug"
+  - OPTS="--buildtype=release"
+  - OPTS="--jerry-lto"
+  - OPTS="--no-snapshot"
```

Comment:
Yes. That's a valid command line argument of `tools/build.py`. It disables the use of the snapshot feature. In this case, it runs a test job to check that variant as well. (Was that what you asked?)